### DISCUSSION TITLE
Languages

### DISCUSSION THREAD

**Discussion by Roxas2712** [victorbalssa/abacus#17] [Ideas]
Hello:) As for now the app is just working great - would it be feasible to add a few more languages? Like German? I could help with that if help is needed ^^.

Best regards

**Comment by victorbalssa** [comment]
Hello @Roxas2712,
Thank you for your support.

I like this idea and I will add it to the feature requests list, I might need your help to validate the translation file when it's done.

**Reply by Roxas2712** [reply]
Great! Of course - just write me and I will support then:)

**Reply by victorbalssa** [reply]
Hello @Roxas2712,
Translation feature is here,
if you want to add the German file you are more than welcome, or I can add it by myself if you provide the translated version of this file: https://github.com/victorbalssa/abacus/blob/updates/src/i18n/locale/translations/en-US.ts

Thank you!
